pset-functions.js

```diff
@@ -136,6 +177,43 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+function scrabbleScore(word){
+const letters = word.split("")
+const onePoint = ["a","e","i","o","u","l", "n", "r", "s", "t"] 
```

Comment:
@Tambini good job coming up with a way to map the letters to their respective score.